plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -296,8 +296,14 @@ func (c *MaxPDVolumeCountChecker) predicate(pod *v1.Pod, meta algorithm.Predicat
 	}
 
 	numNewVolumes := len(newVolumes)
+	node := nodeInfo.Node()
+	instanceType := ""
+	if node != nil {
+		instanceType = node.ObjectMeta.Labels[kubeletapis.LabelInstanceType]
+	}
+	maxVolumes := c.maxPDCount(instanceType)
```

Comment:
Can we pass in Node object instead?  Then each cloud provider can use whatever label or annotation they have to determine the limit.

Reply:
changed to pass a node